src/Compiler/Symbols/Symbols.fs

```diff
@@ -2072,6 +2073,9 @@ type FSharpMemberOrFunctionOrValue(cenv, d:FSharpMemberOrValData, item) =
         | P p ->
             let range = defaultArg sym.DeclarationLocationOpt range0
             match GetXmlDocSigOfProp cenv.infoReader range p with
+            | Some (_, docsig) when p.IsFSharpEventProperty && docsig.StartsWith("P:") -> 
+                // For CLIEvent properties, use E: prefix instead of P:
+                "E:" + docsig.Substring(2)
```

Comment:
Can this code have any unintended consequences when it comes to IDE tooling?
Are all places that work with XmlDocSig in this full codebase adjusted to behave equally?